attribute-masking-service/src/component-tests/java/uk/gov/gchq/palisade/component/attributemask/web/RestControllerWebMvcTest.java

```diff
@@ -30,7 +30,7 @@
 import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
 
 import uk.gov.gchq.palisade.service.attributemask.ApplicationTestData;
-import uk.gov.gchq.palisade.service.attributemask.model.Token;
+import uk.gov.gchq.palisade.service.attributemask.common.Token;
```

Comment:
There is a Token class in the package model as well as the package common.   One needs to be removed, my vote would be the one in common.

Reply:
updated!